### DISCUSSION TITLE
Cyberduck Crash when click on "Transfers" Icon

### DISCUSSION THREAD

**Discussion by Ballistarius** [iterate-ch/cyberduck#15253] [Q&A]
Hi, I have an issue with Cyberduck, the application start ok but when I click on the "Transfers" icon the application crash with a prompt message "Do you want to report the last Crash?". I will appreciate if anyone can help on this.

![CyberDuck_Error](https://github.com/iterate-ch/cyberduck/assets/148813816/911dce5e-3065-4e81-8039-bb9284fc0f64)

Thanks in advance!

**Comment by AliveDevil** [comment]
Please delete the files in `%AppData%\Cyberduck\Transfers`.

**Reply by Ballistarius** [reply]
Hi, thanks for the reply, I tried that and don't help. 

Regards
